can/logger.py

```diff
@@ -53,10 +54,15 @@ def main():
     parser.add_argument('-b', '--bitrate', type=int,
                         help='''Bitrate to use for the CAN bus.''')
 
+    parser.add_argument('-s', '--state', type=int, dest='state', default=1,
```

Comment:
Instead of the user having to look up or remember that `--state=1` is active and `--state=0` is passive this should instead be a flag (e.g. `--passive`)